change plotfile RZ allowed diag to r,t from x,y (#3663)
ECP-WarpX/WarpX: Docs/source/usage/parameters.rst

    Possible scalar fields: ``part_per_cell`` ``rho`` ``phi`` ``F`` ``part_per_grid`` ``divE`` ``divB`` and ``rho_<species_name>``, where ``<species_name>`` must match the name of one of the available particle species. Note that ``phi`` will only be written out when do_electrostatic==labframe. Also, note that for ``<diag_name>.diag_type = BackTransformed``, the only scalar field currently supported is ``rho``.
    Possible vector field components in Cartesian geometry: ``Ex`` ``Ey`` ``Ez`` ``Bx`` ``By`` ``Bz`` ``jx`` ``jy`` ``jz``.
    Possible vector field components in RZ geometry: ``Er`` ``Et`` ``Ez`` ``Br`` ``Bt`` ``Bz`` ``jr`` ``jt`` ``jz``.

>     The default is ``<diag_name>.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz``,
>     unless in RZ geometry,
>     then the default is ``<diag_name>.fields_to_plot = Er Et Ez Br Bt Bz jr jt jz``.

Reviewer: This could be maybe rephrased (more clearly?) as
```suggestion
    The default is ``<diag_name>.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz`` in Cartesian geometry and ``<diag_name>.fields_to_plot = Er Et Ez Br Bt Bz jr jt jz`` in RZ geometry.
```
